polyfills/HTMLFormElement/prototype/requestSubmit/config.toml

```
@@ -0,0 +1,20 @@
+aliases = []
+dependencies = []
+spec = "https://html.spec.whatwg.org/multipage/forms.html#dom-form-requestsubmit-dev"
+docs = "https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/requestSubmit"
+license = "MIT"
+
+[browsers]
+android = "<76"
+chrome = "<76"
+edge = "<79"
+firefox = "<75"
+firefox_mob = "<79"
+ie = "*"
+ie_mob = "*"
+opera = "<63"
+op_mob = "<64"
+op_mini = "*"
+safari = "<16"
+ios_saf = "<16"
+samsung_mob = "<12"
```

```suggestion
[browsers]
android = "*"
chrome = "<76"
edge = "*"
edge_mob = "*"
firefox = "<75"
firefox_mob = "<79"
ie = "*"
ie_mob = "*"
opera = "<63"
op_mob = "<64"
op_mini = "*"
safari = "<16.0"
ios_saf = "<16.0"
samsung_mob = "<12.0"
```

- we record `edge < 79` as `edge *` (same for `android`)
- `safari`, `ios_saf`, and `samsung_mob` use `major.minor` notation
- `edge_mob` can have the same value as `edge`